client-app/package.json

```diff
@@ -9,7 +9,8 @@
     "test:unit": "vue-cli-service test:unit",
     "test:e2e": "vue-cli-service test:e2e",
     "lint": "vue-cli-service lint",
-    "i18n:report": "vue-cli-service i18n:report --src './src/**/*.?(js|vue)' --locales './src/locales/**/*.json'"
+    "i18n:report": "vue-cli-service i18n:report --src './src/**/*.?(js|vue)' --locales './src/locales/**/*.json'",
+    "nswag:genclient": "nswag run"
```

Comment:
Are you sure we need it? Looks like `nswag run` is shorter than `npm run nswag:genclient`

Reply:
It is not except this usage. In such case  'npx nswag run' if nswag is not installed globally. We can cut package.json to more shortly view, for example just 'npm run nswag'.